src/api.json

```
@@ -93,6 +93,18 @@
       "address": "string"
     }
   },
+  "light/get_aas_by_base_aas": {
+    "params": {
+      "base_aa": "object",
```

Here and the others `base_aas`, `aa` and `aas` also should be changed to string like this: 
```suggestion
      "base_aa": "string",
```